Gaijinizer patch/Map003.txt

```diff
@@ -1503,50 +1503,54 @@ the number of registrations.#MTLed
  本来なら、気力を最大まで強化しないと厳しいが
 \\C[2]今回は百鬼のアイテムのおかげで、初期気力でも使える。
 > CONTEXT: Map003/events/1/pages/2/389/Dialogue/クッション君
-And this one is stupid for consumption vitality.
-Originally, it is tough if you do not strengthen
-the vitality to the maximum. \\C[2]This time, thanks to
-Hyakki's items, you can use the initial vitality
-as well.#MTLed
+Also, the vitality consumption for this one is
+stupid high. Ordinarily, it would be difficult if
+you didn't upgrade vitality to the maximum, but...
+
+\\C[2]This time, thanks to Hyakki's items, it can
+be used even with the base amount of vitality.#Strate
 > END STRING
 
 > BEGIN STRING
 一度、体験してみるといいぜ。
 
  多分、性欲上昇の値にビックリするだろうから。
 > CONTEXT: Map003/events/1/pages/2/394/Dialogue/クッション君
-You should try to experience it once.
-Maybe because you will be surprised at the value
-of lust rise.#MTLed
+You should try to experience it at least once.
+
+Because you may be surprised at the amount of
+lust buildup.#Strate
 > END STRING
 
 > BEGIN STRING
 ティタノボアは、到底お前が敵うような相手じゃない
 
 見つかったら、逃げるが勝ちだ。
 > CONTEXT: Map003/events/1/pages/2/400/Dialogue/クッション君
-Titanoboa is by far not your opponent.
-If found, it runs away but it wins.#MTLed
+You're no match for Titanoboa at all.
+
+If you're found, you win by running away.#StrateUnsure
```

Comment:
ティタノボアは、到底お前が敵うような相手じゃない = Titanoboa isn't an unbeatable opponent

逃げるが勝ちだ。= means something to the effect of "Discretion is the better part of valor" 

So what I think he's saying is, "Titanoboa isn't unbeatable because escaping is a kind of victory."

Reply:
Makes sense. I reworded it a little to make it sound a bit more natural.